src/libcore/ptr.rs

```diff
@@ -512,6 +512,8 @@ impl<T: ?Sized> PartialOrd for *mut T {
 #[unstable(feature = "unique", reason = "needs an RFC to flesh out design")]
 pub struct Unique<T: ?Sized> {
     pointer: NonZero<*const T>,
+    // NOTE: this marker has no consequences for variance, but is necessary
+    // for dropck to understand that we logically contain a `T`.
```

Comment:
(having said that, I have no problem with using the word "contain" here.  The one thing you might do is link to the relevant part of RFC, namely: https://github.com/rust-lang/rfcs/blob/master/text/0769-sound-generic-drop.md#phantom-data )